pyOCD/tools/flash_tool.py

```diff
@@ -77,7 +86,8 @@
 group.add_argument("-se", "--sector_erase", action="store_true", help="Use sector erase when programming.")
```

Comment:
Originally sector_erase and chip_erase were meant to direct the method for program flash more than specifying and actual chip erase or page erase.  Right now page erase programming won't actually erase any pages if the data hasn't changed.  Maybe a new parameter -ft --flash-type <chiperase, pageerase, auto> could be used for programming instead of these.  This would remove some ambiguity.